create-loop/src/index.ts

```diff
@@ -1,14 +1,38 @@
 #!/usr/bin/env node
 
 /* eslint-disable @typescript-eslint/no-var-requires */
-const fs = require('fs/promises');
-const path = require('path');
-const { execSync } = require('child_process');
-const prompts = require('prompts');
-const Sqrl = require('squirrelly');
-const { default: templates } = require('@oliveai/loop-templates');
-
-const projectOptions = {
+import * as fs from 'fs/promises';
+import { execSync } from 'child_process';
+import * as path from 'path';
+import { platform } from 'process';
+import * as prompts from 'prompts';
+import * as Sqrl from 'squirrelly';
+import * as ua from 'universal-analytics';
+import { default as templates } from '@oliveai/loop-templates';
+import { TemplatesObject, TemplateFile } from '@oliveai/loop-templates/dist/types';
+
+const analytics = __GOOGLE_ANALYTICS_ID__ ? 
+  ua(__GOOGLE_ANALYTICS_ID__) : 
+  // For development
+  {
+    event: () => ({
+        send: () => undefined
+      }),
+    set: () => undefined,
+  };
+
+const osName = platform === 'win32' ? 'windows' : platform;
```

Comment:
May I know why we specify win32 here? 🤔

Reply:
like why we set cd1 for all other environments and cd4 only for windows?

Reply:
`cd1` is for the environment, e.g. "dev" vs "prod"

`cd4` is for the OS name. For some reason we specify Windows as `win32` in the rest of our analytics for Helps, so I'm just following what they did :)

Reply:
https://gitlab.oliveai.com/oliveai/platform/desktop/ui/-/blob/main/src/main/analytics/index.ts#L69 Here's where UI does the same if you're interested! :)

Reply:
Ahhh thx for the explanation!